### DISCUSSION TITLE
Project 1

### DISCUSSION THREAD

**Discussion by UniquelyAbbi** [cooperrc/computational-mechanics#29] [General]
I just started working on the project. We can discuss about it here if you'd like. Ill start off with needing an explanation on what I did incorrectly and we see how we'll do.

![image](https://user-images.githubusercontent.com/106557922/172020908-cf73e317-f21f-4f07-a7df-3c86158ea3ab.png)

I am reading the third chapter in module 3 i'm not sure if that may be helpful, there is a section that talks about finite difference

**Comment by ecieszynski** [comment]
I dont think you can initialize variables as T(t), etc because it assumes that you're calling a function called T with the parameter t. Think of it when you use print(), you're calling the print function and it will use the given parameters to execute the print algorithm, if that makes sense

**Reply by UniquelyAbbi** [reply]
I put find value K in the second question but it isnt working at the end, says invalid syntax. I am going to try just find k and see.
